server/data/alertsApiClient.ts

```diff
@@ -0,0 +1,14 @@
+import config from '../config'
+import { AlertListResponseDataItem } from '../@types'
+
+export default class AlertsApiClient {
+  async getAlerts(): Promise<AlertListResponseDataItem[]> {
+    const alertManagerEndpoint = `${config.apis.alertManager.url}/alerts`
+    const urlFilter = 'filter=businessUnit="hmpps"'
+    return fetch(`${alertManagerEndpoint}?${urlFilter}`)
```

Comment:
We don't use `fetch`, have a nosey at the StrapiApiClient, we use a standard `RestClient` and you should just be able to copy that for the most part 🙂

Reply:
Incidentally `async` `await` is preferable to `.then()` etc. Easier to understand and follow and can be wrapped in `try/catch`